cmake/WaveUtils.cmake

```diff
@@ -40,6 +40,31 @@ FUNCTION(WAVE_ADD_TEST NAME)
 
 ENDFUNCTION(WAVE_ADD_TEST)
 
+# wave_add_benchmark: Add a target which links against google benchmark
+#
+# WAVE_ADD_BENCHMARK(Name src1 [src2...])
```

Comment:
@leokoppel, Do you mind renaming this file? We briefly talked about this on PR #226.

Reply:
Okay I see #233. Will approve this then.